tests/http_proxy_io_ut/http_proxy_io_ut.c

```diff
@@ -1575,6 +1575,70 @@ TEST_FUNCTION(http_proxy_io_retrieve_options_calls_the_underlying_retrieve_optio
     http_proxy_io_get_interface_description()->concrete_io_destroy(http_io);
 }
 
+#define OPTION_UNDERLYING_IO_OPTIONS "underlying_io_options"
+
+/* GitHub PR 564 */
```

Comment:
these labels necessary?

Reply:
Oh, the label yes, but the define should be moved to the top. Thanks.